code/game/objects/items/rogueitems/keyrings.dm

```diff
@@ -125,19 +125,19 @@
 	keys = list(/obj/item/key/sheriff_office, /obj/item/key/town_armory, /obj/item/key/town_barracks, /obj/item/key/town_dungeon)
 
 /obj/item/storage/keyring/man_at_arms
-	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/manor, /obj/item/key/walls)
+	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/manor, /obj/item/key/walls, /obj/item/key/keep_armory)
 
-/obj/item/storage/keyring/squire 
+/obj/item/storage/keyring/squire
 	keys = list(/obj/item/key/keep_gatehouse, /obj/item/key/keep_barracks, /obj/item/key/manor, /obj/item/key/walls)
 
 /obj/item/storage/keyring/captain
-	keys = list(/obj/item/key/councillor_rooms, /obj/item/key/keep_gatehouse, /obj/item/key/keep_armory, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/manor, /obj/item/key/walls)
+	keys = list(/obj/item/key/councillor_rooms, /obj/item/key/keep_gatehouse, /obj/item/key/keep_armory, /obj/item/key/keep_barracks, /obj/item/key/keep_dungeon, /obj/item/key/keep_dungeon_torture, /obj/item/key/manor, /obj/item/key/walls)
```

Comment:
I'm a tad skeptical in giving the captain torture room keys as well.
We've had issues in the past with people intentionally interrupting the dungeoneer in whatever they do for white knight purposes, and making that easier to be done isn't a good idea, IMO.